fk/luse/src/main/java/se/inera/intyg/common/luse/v1/model/converter/RespConstants.java

```diff
@@ -0,0 +1,130 @@
+    public static final String GRUNDFORMEDICINSKTUNDERLAG_BESKRIVNING_DELSVAR_JSON_ID_1 = "annatGrundForMUBeskrivning";
+    public static final String MOTIVERING_TILL_INTE_BASERAT_PA_UNDERLAG_ID_1 = "motiveringTillInteBaseratPaUndersokning";
+    public static final String GRUNDFORMEDICINSKTUNDERLAG_SVAR_TEXT_ID = "FRG_1.RBK";
+    public static final String GRUNDFORMEDICINSKTUNDERLAG_ANNANBESKRIVNING_DELSVAR_TEXT = "DFR_1.3.RBK";
+    public static final String KANNEDOM_SVAR_ID_2 = "2";
+    public static final String KANNEDOM_SVAR_TEXT_ID = "FRG_2.RBK";
+    public static final String KANNEDOM_SVAR_JSON_ID_2 = "kannedomOmPatient";
+    public static final String UNDERLAGFINNS_SVAR_ID_3 = "3";
+    public static final String UNDERLAGFINNS_SVAR_JSON_ID_3 = "underlagFinns";
+    public static final String UNDERLAG_SVAR_ID_4 = "4";
+    public static final String UNDERLAG_SVAR_JSON_ID_4 = "underlag";
+    public static final String MOTIVERING_TILL_INTE_BASERAT_PA_UNDERLAG_DELSVAR_ID_1 = "1.4";
+    public static final String MOTIVERING_TILL_INTE_BASERAT_PA_UNDERLAG_TEXT = "Motivering till varför det medicinska underlaget "
+        + "inte baseras på en undersökning av patienten:";
+    public static final String MOTIVERING_TILL_INTE_BASERAT_PA_UNDERLAG_DESCRIPTION_ID = "FRG_25.RBK";
+    public static final String MOTIVERING_TILL_INTE_BASERAT_PA_UNDERLAG_DESCRIPTION = "Detta är ett stöd för ifyllnad och ingen fråga"
+        + " från Försäkringskassan. Informationen du anger här kan vara till hjälp för Försäkringskassan i deras handläggning. "
+        + "Informationen du anger nedan, kommer att överföras till fältet \"{0}\" vid signering.";
+    public static final String GRUNDFORMU_JOURNALUPPGIFTER_LABEL = "KV_FKMU_0001.JOURNALUPPGIFTER.RBK";
+    public static final String GRUNDFORMU_ANHORIG_BESKRIVNING_LABEL = "KV_FKMU_0001.ANHORIG.RBK";
+    public static final String GRUNDFORMU_UNDERSOKNING_LABEL = "KV_FKMU_0001.UNDERSOKNING.RBK";
+    public static final String GRUNDFORMU_ANNAT_LABEL = "KV_FKMU_0001.ANNAT.RBK";
+    public static final String UNDERLAGFINNS_SELECTED_TEXT = "Ja";
+    public static final String UNDERLAGFINNS_UNSELECTED_TEXT = "Nej";
+    public static final String UNDERLAGFINNS_SVAR_TEXT = "FRG_3.RBK";
+    public static final String UNDERLAG_TYPE_TEXT_ID = "FRG_4.RBK";
+    public static final String UNDERLAG_DATUM_TEXT = "Datum";
+    public static final String UNDERLAG_INFORMATION_SOURCE_TEXT_ID = "DFR_4.3.RBK";
+    public static final String UNDERLAG_INFORMATION_SOURCE_DESCRIPTION_ID = "DFR_4.3.HLP";
+
+    public static final String DIAGNOS_CATEGORY_ID = "diagnos";
+    public static final String DIAGNOS_CATEGORY_TEXT_ID = "KAT_4.RBK";
+    public static final String DIAGNOS_SVAR_TEXT_ID = "FRG_6.RBK";
+    public static final String DIAGNOS_SVAR_ID = "6";
+    public static final String DIAGNOS_ICD_10_ID = "ICD_10_SE";
```

Comment:
A misstake, originally made by me, when using the abstract definition of diagnos. The AbstractQuestionDiagnoser uses constants defined in the fkparent and it would be a misstake for us to have the same constant in multiple places. 

I know we move constants to the specific certificate type and version, but for constants used by fkparent definitions should (I think) only be defined on that level as well. 

What do you think about removing the constants in RespConstants for the fk-certificate types that are using the abstract definition? That means only the constants directly used by AbstractQuestionDiagnoser.

Reply:
I agree! Done.